docs/install/installation.md

```diff
@@ -19,7 +19,7 @@ Argo CD is running. Don't worry, without any configuration, it will not start me
     We also provide a Kustomize base in addition to the plain Kubernetes YAML
     manifests. You can use it as remote base and create overlays with your
     configuration on top of it. The remote base's URL is
-    `https://github.com/argoproj-labs/argocd-image-updater/manifests/base`
+    `https://github.com/argoproj-labs/argocd-image-updater/tree/master/manifests/base`
 
```

Comment:
Maybe we can use the stable version, to be consistent with the `install.yaml` later in this doc:
https://github.com/argoproj-labs/argocd-image-updater/tree/stable/manifests/base

Reply:
Correct!